src-ts/lib/styles/_layout.scss

```diff
@@ -1,6 +1,7 @@
 @import 'palette';
 
 $header-height: 80px;
+$max-content-width: 1440px;
```

Comment:
plz use the $xxl-min: 1440px; variable in the _breakpoints.scss file

Reply:
@brooketopcoder like $max-content-width: $xxl-min, or should I use directly $xxl-min?

Reply:
@vas3a i think just use the variable directly--no need for a duplicate